lib/hook.template.raw

```diff
@@ -1,15 +1,53 @@
 #!/usr/bin/env node
 // {{generated_message}}
 
-(function hookEntryPoint() {
+const fs = require('fs')
+const path = require('path')
+
+;(function hookEntryPoint() {
```

Comment:
Hmm... if there was initially, I guess it has become irrelevant in the mean while ...
Will get rid of it.